### ISSUE TITLE
Midterm Peer Review (jcv67)

### ISSUE THREAD

**Issue by skydaddy7419** [amg369/ORIE_4741_Project#8]
As an avid music listener, I am quite interested in the results of this project, questioning songs that are characterized into four groups; happy, sad, angry, and relaxed. With these four subsets, the group then plans to analyze the lyrics of each individual song to find if a model could be generated that looks at the lyrics of a song and predict what theme of the song it is. The group did a great job of removing unnecessary text in the lyrics and displaying a Naive Bayes model and a correlation matrix.

Compliments:
1. Choosing to analyze the correlation of popular words is a great initial step,  as the group was able to find that the popular words were generally independent of one another, however, this may not be the case for not as commonly used words.
2. Analyzing the Naive Bayes model on its effectiveness in this case and the validation of where to split the data was a great addition. Finding the hurdles in your project and solving them shows sufficient research into the case.
3. I am intrigued by how a loss function will affect the model, and adding the factor may add confidence in the model generated.

Recommendations:
1. An interesting question I have is whether this is correlation of moods with sentences or small combinations of words. This could be analternative perspective to investigate.
2. I would like to see more graphs to visualize the data collected. The ending graphs were interesting, however, were not to correlated with the information presented before them.
3. I hope the neural network model is sufficient for your data analysis, however, I am doubtful it will account for individual features at a lesser importance than a supervised learning model would.
